spec/generators/rspec/install/install_generator_spec.rb

```diff
@@ -27,7 +27,7 @@ def require_rspec_rails
     match(/^require 'rspec\/rails'$/m)
   end
 
-  def use_transactional_fixtures
+  def have_transactional_fixtures
```

Comment:
Maybe `enabled_transactional_fixtures?`? This is detecting the feature, not enabling it, right?